docs/standard/microservices-architecture/implement-resilient-applications/implement-custom-http-call-retries-exponential-backoff.md

```diff
@@ -107,9 +107,9 @@ public async Task<Catalog> GetCatalogItems(int page,int take, int? brand, int? t
 }
 ```
 
-However, this code is suitable only as a proof of concept. The next topic explains how to use more sophisticated and proven libraries.
+However, and as mentioned, this code is suitable only as a proof of concept. The next topic explains how to use more sophisticated approaches while simpler to use with HttpClientFactory, available since .NET Core 2.1, plus proven libraries like Polly.
```

Comment:
+However, and as mentioned, -> Remember that

next topic -> next article

Last sentence is too long. Break it down similar to what I showed earlier.

Reply:
👍